lib/dialects/sqlite/query-generator.js

```diff
@@ -428,12 +428,14 @@ class SQLiteQueryGenerator extends MySqlQueryGenerator {
     ).join(', ');
     const attributeNamesExport = Object.keys(attributes).map(attr => this.quoteIdentifier(attr)).join(', ');
 
-    return this.createTableQuery(backupTableName, attributes).replace('CREATE TABLE', 'CREATE TEMPORARY TABLE')
+    return 'PRAGMA foreign_keys=off;'
```

Comment:
We should wrap whole rename column queries under `PRAGMA foreign_keys=off|on;` not just the create table.

Reply:
Sorry, I am confused. Did not I implement it like you said?

The return of this function will wrapped like this
```
PRAGMA foreign_keys=off;
exist query....
PRAGMA foreign_keys=on;
```

Reply:
Right, I think I got mislead by `createTableQuery` :)